tox.ini

```diff
@@ -10,12 +10,26 @@ deps =
     graphql-core>=2.1
     pytest-cov
 commands =
-    py{py,27,34,35,36}: py.test tests {posargs}
+    py{py,27,34,35,36}: py.test --cov=graphql_server tests {posargs}
+
+[testenv:pre-commit]
+basepython=python3.6
+deps =
+    pre-commit>0.12.0
+setenv =
+    LC_CTYPE=en_US.UTF-8
```

Comment:
what are these 3 environment vars for?

Reply:
So graphene has the locale set for en US I, I believe, however when running on a system that handles the others (which are preferred) black has issues. I'll dig out the error I had tomorrow that made these go away